src/descriptor/template.rs

```diff
@@ -466,6 +472,40 @@ mod test {
     use miniscript::descriptor::{DescriptorPublicKey, DescriptorTrait, KeyMap};
     use miniscript::Descriptor;
 
+    // BIP44 `pkh(key/44'/{0,1}'/0'/{0,1}/*)`
+    #[test]
+    fn test_bip44_template_cointype() {
```

Comment:
I feel like this test can be generalized to include cointype checking for all the templates 44, 49 and 84. Can be done in a single test. and can be named `test_template_cointype()`.

Reply:
I agree :)

Reply:
sure, I can take care of that